modules/farm_fd2_school/src/entrypoints/FD3_Application/App.vue

```diff
@@ -11,24 +11,17 @@
       v-model:date="date"
     />
     <br />
-    <label
-      for="harvest-crop"
-      class="label-margin"
-    >
-      Crop:
-    </label>
-    <select
-      id="harvest-crop"
-      v-model="crop"
-    >
-      <option
-        v-for="crop in cropList"
-        v-bind:key="crop.id"
-        v-bind:value="crop"
-      >
-        {{ crop.attributes.name }}
-      </option>
-    </select>
+    <CropSelector
+      v-bind:required="true"
+      v-bind:showValidityStyling="validity.showStyling"
```

Comment:
Since you do not use `validity.showStyling` or `validity.crop` in any significant way in the `script` they could have been omitted here. However, if you wanted to use them the value of `validity.crop` could have been used in the `formValid` computed property and that property could have set `validity.showStyling`. That way the component itself gets to determine if the value is valid or not.